frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "code-review-frontend",
-  "version": "1.8.5",
+  "version": "1.8.6",
```

Comment:
Could you also update package-lock.json (manually to just update the version number)